pkg/install/stack/usercluster-mla/stack.go

```diff
@@ -0,0 +1,387 @@
+	sublogger := log.Prefix(logger, "   ")
+
+	chart, err := helm.LoadChart(filepath.Join(opt.ChartsDirectory, UserClusterMlaChartsPrefix, AlertmanagerProxyChartName))
+	if err != nil {
+		return fmt.Errorf("failed to load Helm chart: %w", err)
+	}
+
+	if err := util.EnsureNamespace(ctx, sublogger, kubeClient, AlertmanagerProxyNamespace); err != nil {
+		return fmt.Errorf("failed to create namespace: %w", err)
+	}
+
+	release, err := util.CheckHelmRelease(ctx, sublogger, helmClient, AlertmanagerProxyNamespace, AlertmanagerProxyReleaseName)
+	if err != nil {
+		return fmt.Errorf("failed to check to Helm release: %w", err)
+	}
+
+	if err := util.DeployHelmChart(ctx, sublogger, helmClient, chart, AlertmanagerProxyNamespace, AlertmanagerProxyReleaseName, opt.HelmValues, true, opt.ForceHelmReleaseUpgrade, opt.DisableDependencyUpdate, release); err != nil {
+		return fmt.Errorf("failed to deploy Helm release: %w", err)
+	}
+
+	logger.Info("✅ Success.")
+
+	return nil
+}
+
+func deployConsul(ctx context.Context, logger *logrus.Entry, kubeClient ctrlruntimeclient.Client, helmClient helm.Client, opt stack.DeployOptions) error {
+	logger.Info("📦 Deploying Consul…")
+	sublogger := log.Prefix(logger, "   ")
+
+	chart, err := helm.LoadChart(filepath.Join(opt.ChartsDirectory, UserClusterMlaChartsPrefix, ConsulChartName))
+	if err != nil {
+		return fmt.Errorf("failed to load Helm chart: %w", err)
+	}
+
+	if err := util.EnsureNamespace(ctx, sublogger, kubeClient, ConsulNamespace); err != nil {
+		return fmt.Errorf("failed to create namespace: %w", err)
+	}
+
+	release, err := util.CheckHelmRelease(ctx, sublogger, helmClient, ConsulNamespace, ConsulReleaseName)
+	if err != nil {
+		return fmt.Errorf("failed to check to Helm release: %w", err)
+	}
+
+	if err := util.DeployHelmChart(ctx, sublogger, helmClient, chart, ConsulNamespace, ConsulReleaseName, opt.HelmValues, true, opt.ForceHelmReleaseUpgrade, opt.DisableDependencyUpdate, release); err != nil {
+		return fmt.Errorf("failed to deploy Helm release: %w", err)
+	}
+
+	logger.Info("✅ Success.")
+
+	return nil
+}
+
+func deployCortex(ctx context.Context, logger *logrus.Entry, kubeClient ctrlruntimeclient.Client, helmClient helm.Client, opt stack.DeployOptions) error {
+	logger.Info("📦 Deploying Cortex…")
+	sublogger := log.Prefix(logger, "   ")
+
+	chart, err := helm.LoadChart(filepath.Join(opt.ChartsDirectory, UserClusterMlaChartsPrefix, CortexChartName))
+	if err != nil {
+		return fmt.Errorf("failed to load Helm chart: %w", err)
+	}
+
+	if err := util.EnsureNamespace(ctx, sublogger, kubeClient, CortexNamespace); err != nil {
+		return fmt.Errorf("failed to create namespace: %w", err)
+	}
+
+	release, err := util.CheckHelmRelease(ctx, sublogger, helmClient, CortexNamespace, CortexReleaseName)
+	if err != nil {
+		return fmt.Errorf("failed to check to Helm release: %w", err)
+	}
+
+	runtimeConfigMap := &corev1.ConfigMap{
+		Data: map[string]string{"runtime-config.yaml": ""},
```

Comment:
could we use the constants from `/pkg/controller/seed-controller-manager/mla/ratelimit_cortex_controller.go` here (and also 2 lines below)?

Reply:
Done!